oracle/README.md

```diff
@@ -180,10 +180,12 @@ where the _watcher_ could be one of:
 | `HOME_GAS_PRICE_SPEED_TYPE` | Assuming the gas price oracle responds with the following JSON structure: `{"fast": 20.0, "block_time": 12.834, "health": true, "standard": 6.0, "block_number": 6470469, "instant": 71.0, "slow": 1.889}`, this parameter specifies the desirable transaction speed. The speed type can be omitted when `HOME_GAS_PRICE_ORACLE_URL` is not used. | `instant` / `fast` / `standard` / `slow` |
 | `HOME_GAS_PRICE_FALLBACK` | The gas price (in Wei) that is used if both the oracle and the fall back gas price specified in the Home Bridge contract are not available. | integer |
 | `HOME_GAS_PRICE_UPDATE_INTERVAL` | An interval in milliseconds used to get the updated gas price value either from the oracle or from the Home Bridge contract. | integer |
+| `HOME_GAS_PRICE_FACTOR` | A value that will multiply the gas price of the oracle to convert it to gwei. If the oracle API returns gas prices in gwei then this can be set to `1`. | integer |
```

Comment:
Please extend the description to point out that this factor could be used to intentionally pay more gas than suggested by the oracle in order to guarantee the transaction verification in the moment of sudden growth of gas price. E.g. `1.25` or `1.5` could be still fine in this case.

Reply:
Updated description, please check https://github.com/poanetwork/tokenbridge/pull/142/commits/f7bb0a869e9d18388a5ff701425bc6dd3b2f5ca0